environment_linux_cpuonly.yml

```diff
@@ -15,6 +15,7 @@ dependencies:
   - isort
   # dask and related
   - dask
+  - dask.distributed 
```

Comment:
this is causing the CI failure, its just distributed: https://anaconda.org/conda-forge/distributed